### DISCUSSION TITLE
Need help newbie here

### DISCUSSION THREAD

**Discussion by Jayke770** [socketio/socket.io#4070] [Q&A] [answered]
How to include a separate namespace using a separate js file or module 

for example: 
`const express = require('express')`
`const app = express()` 
`const http = require('http').createServer(app)`
`const io = require('socket.io')(http)` 
`const port = process.env.PORT || 8989` 
`const namespace = require('admin')`

Then how to separate the namespace in a different file or module 

`http.listen(port, console.log('Server Started on port ' + port))`

**Comment by darrachequesne** [comment, marked as answer]
Hi! Did you check https://socket.io/docs/v4/server-application-structure/ ?

**Reply by Jayke770** [reply]
thank you sir 😃
